app/navigation/CharityInformationNavigator.scala

```diff
@@ -64,7 +68,7 @@ class CharityInformationNavigator @Inject()(implicit frontendAppConfig: Frontend
       case _ => routes.SessionExpiredController.onPageLoad()
     }
     case CharityOfficialAddressLookupPage => userAnswers: UserAnswers => userAnswers.get(CharityOfficialAddressLookupPage) match {
-      case Some(_) => charityInfoRoutes.CharityInformationSummaryController.onPageLoad()
+      case Some(_) => charityInfoRoutes.CanWeSendToThisAddressController.onPageLoad(CheckMode)
       case _ => routes.SessionExpiredController.onPageLoad()
     }
     case CanWeSendToThisAddressPage => userAnswers: UserAnswers => userAnswers.get(CanWeSendToThisAddressPage) match {
```

Comment:
This change route will need matching to the normalMode routes to be able to accommodate routing to the postal address if the user goes from the summary page to the "CanWeSendToThisAddressPage" and selects "No". This will stop the user from being able to get to the summary page without entering a postal address (Please see below)
<img width="716" alt="Screenshot 2020-08-12 at 10 32 58" src="https://user-images.githubusercontent.com/28566263/90003348-d62c1700-dc8b-11ea-9327-65281e7f5359.png">